api/sixtyfps-cpp/docs/cpp_migration.md

```diff
@@ -29,3 +29,27 @@ New code:
 sixtyfps::Value args[] = { SharedString("Hello"), 42. };
 instance->invoke_callback("foo", args);
 ```
+
+#### Models
+
+`Model::row_data` returns now a `std::optional<ModelData>` and can thus be used with indices that are out of bounds.
+
+This also means that `Model`s must handle invalid indices and may not crash when a invalid index is passed in.
+
+Old code:
+
+```cpp
+another_model->row_data(2);
```

Comment:
I think it would be a good idea to use the return value in a way that is similar to the "new" code.

```suggestion
float value = another_model->row_data(2);
do_something(value);
```